src/Login.ts

```diff
@@ -174,9 +174,11 @@ export class Login {
     } catch (e) {
       //
     }
-    Login.loginWithRedirect(params);
-    // @ts-ignore (will do a browser redirect so nothing to care about)
-    return;
+
+    // Login.authorize should never resolve on browser redirect (to avoid return value undefined)
+    return new Promise(() => {
```

Comment:
Will client receive Promise instead of undefined so it should still process it?

Reply:
Yes, but they will receive a promise anyway (async function).